File: ConfiEscola/ClientApp/src/app/viewModels/historicoEscolarViewModel.ts
export class historicoEscolarViewModel{
    id:number
    formato: string
    nome: string
    arquivo: any

    constructor(id: number, formato: string, nome: string)
    {   
        this.id=id;
        this.formato=formato;
        this.nome=nome;
    }

    public setArquivo(arquivo:any)
    {
        this.arquivo=arquivo;
    }
}

File: ConfiEscola.Application/ViewModels/HistoricoEscolarViewModel.cs
using ConfiEscola.Domain.Models;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace ConfiEscola.Application.ViewModels
{
    public class HistoricoEscolarViewModel
    {
        public int Id { get; set; }
        public string Formato { get; set; }
        public string Nome { get; set; }
        public byte[] Arquivo { get; set; }
    }
}